src/myr/reference.js

```diff
@@ -363,6 +363,26 @@ let transformations = [
         description: <span>The setPhiLength function sets the phi length of the cursor in degrees. This changes the number of degrees shown of certain shapes. Interesting shapes can be achieved with values greater than 360. The default phi length is 360.</span>,
         example: "setPhiLength"
     },
+    {
+        name: "randomInt",
+        parameters: [{ type: "number", name: "min" }, { type: "number", name: "max" }],
+        description: <span>The randomInt function generates a random integer derived from the current seed. The range begins with the minimum number and includes every integer less than the maximum. The default range is -40 to 40.</span>,
+    },
+    {
+        name: "random",
+        parameters: [{ type: "number", name: "min" }, { type: "number", name: "max" }],
+        description: <span>The random function generates a random float derived from the current seed. The range begins with the minimum number and includes every number less than the maximum. The default range is -40 to 40.</span>,
+    },    
+    {
+        name: "setSeed",
+        parameters: [{ type: "number", name: "seed" }],
+        description: <span>The setSeed function sets the seed used by the randomInt and random functions to generate random numbers. The default value is based on the current time.</span>,
+    },
+    {
+        name: "getSeed",
+        parameters: [],
+        description: <span>The getSeed function returns the value of the current seed.</span>,
```

Comment:
We should make sure that our descriptions don't say the obvious. There should be more information about this.

Reply:
Changing it to "The getSeed function returns the value of the current seed used by the randomInt and random functions to generate random numbers." I'm not sure what else to add that would be helpful without going into a long explanation of what seeding is.